packages/web-shared/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Can you also symlink the LICENSE from the repo root please like we do in all the other packages?